boringtun/Cargo.toml

```diff
@@ -29,6 +29,8 @@ blake2 = "0.10"
 hmac = "0.12"
 jni = { version = "0.19.0", optional = true }
 thiserror = "1.0"
+rand = "0.8"
+tokio = { version = ">=1.22", features = ["rt", "net", "sync", "macros"] }
```

Comment:
I think this is not needed in the library, only in the tests. If so please move this to the dev dependencies section

Reply:
Also moved the testcase into integration_tests module